src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, combineReducers } from 'redux';
+import categoriesReducer from './categories/categories';
+import booksReducer from './books/books';
+import visibilityReducer from './filter/filter';
+
+const rootReducer = combineReducers({
+  books: booksReducer,
+  categories: categoriesReducer,
+  visibility: visibilityReducer,
+});
```

Comment:
- [ ] Kindly remove this reducer. It would be nice to stick to project requirements. For this project, the only reducers needed are `booksReducer` and `categoriesReducer`. I understand you want to add other features, I would advise you to add them after the last milestone of the project. Kindly remove it. 👍

Reply:
All good now 👍🏽